keeweb/appinfo/app.php

```diff
@@ -14,6 +14,14 @@
 use OCP\AppFramework\App;
 use OCA\Keeweb\Controller\PageController;
 
+$mimeTypeDetector = \OC::$server->getMimeTypeDetector();
+$mimeTypeDetector->registerType('kdbx', 'application/x-kdbx', 'application/x-kdbx');
+
+if (\OC::$REQUESTEDAPP === 'dav') {
```

Comment:
cc @rullzer @nickvergessen does that make sense? ;) For a dav request we to announce the mime type to nextcloud. The app itself is not required. I'm trying to save some resources by skipping the app initialization.